src/static/helpers/slasHelper.ts

```diff
@@ -230,9 +244,15 @@ export async function loginRegisteredUserB2C(
     },
   };
 
-  const response = await slasClient.authenticateCustomer(options, true);
+  const response = await slasClientCopy
+    .authenticateCustomer(options, true)
+    .catch((error: Error) => {
```

Comment:
We seem to be just wrapping one error into another here.  We want to check for any client or server errors, for which we'll need to look at the status inside the response.